ui/cap-react/src/antd/util/CodeViewer.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useRef } from "react";
+import { basicSetup } from "codemirror";
+import { EditorState } from "@codemirror/state";
+import { EditorView } from "@codemirror/view";
+import { json } from "@codemirror/lang-json";
+
+const CodeViewer = ({
+  value,
+  lang = json,
```

Comment:
should we have json by default? might break functionality for other extensions, etc